### PR TITLE
Makefile.am files indentation

### LINKED ISSUE #7251: Be consistent in use of tab/space in Makefile.am files
#### Bug report summary
As per our coding standards, the use of tab/spaces in Makefile.am files should be consistent

##### OS / Environment
N/A

##### Netdata version (ouput of `netdata -V`)
v1.18.1

##### Component Name
autotools

##### Steps To Reproduce
Some Makefile.am files use both tabs and whitespaces for indentation

##### Expected behavior
All Makefile.am files should use tabs for indentation of recipe lines and 4 spaces for indentation of non-recipe lines

### PR DESCRIPTION
**Pull request by knatsakis**
##### Summary
Be consistent in the use of tabs/spaces in Makefile.am files

##### Component Name
autotools

##### Additional Information
Use 4 spaces for indentation of non-recipe lines in Makefile.am files

Fixes #7251

### CODE REVIEW

File: collectors/python.d.plugin/gearman/Makefile.inc [hunk 1]
```diff
@@ -10,3 +10,4 @@ dist_pythonconfig_DATA += gearman/gearman.conf
 
 # do not install these files, but include them in the distribution
 dist_noinst_DATA       += gearman/README.md gearman/Makefile.inc
+
```
Comment (amoss) [inline collectors/python.d.plugin/gearman/Makefile.inc]: This new blank is unnecessary

File: collectors/python.d.plugin/nvidia_smi/Makefile.inc [hunk 2]
```diff
@@ -10,3 +10,4 @@ dist_pythonconfig_DATA += nvidia_smi/nvidia_smi.conf
 
 # do not install these files, but include them in the distribution
 dist_noinst_DATA       += nvidia_smi/README.md nvidia_smi/Makefile.inc
+
```
Comment (amoss) [inline collectors/python.d.plugin/nvidia_smi/Makefile.inc]: This new blank is unnecessary

File: health/notifications/hangouts/Makefile.inc [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 # install these files
 dist_noinst_DATA += \
-	hangouts/README.md \
-	hangouts/Makefile.inc \
-	$(NULL)
+    hangouts/README.md \
+    hangouts/Makefile.inc \
+    $(NULL)
+
```
Comment (amoss) [inline health/notifications/hangouts/Makefile.inc]: This new blank is unnecessary.

File: Makefile.am [hunk 4]
```diff
@@ -550,7 +550,7 @@ if ENABLE_PLUGIN_APPS
     plugins_PROGRAMS += apps.plugin
     apps_plugin_SOURCES = $(APPS_PLUGIN_FILES)
     apps_plugin_LDADD = \
-    	$(NETDATA_COMMON_LIBS) \
+        $(NETDATA_COMMON_LIBS) \
         $(OPTIONAL_LIBCAP_LIBS) \
```
Comment (thiagoftsm) [inline Makefile.am]: This line and the next have wrong tabulation yet.

Comment (thiagoftsm) [reply Makefile.am]: Sorry, my fault, it is correct!

**Pull request merged**